.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: ruby
+
+rvm:
+  - 2.4.3
+  - 2.5.0
```

Comment:
Does this seem like a reasonable support matrix? Or should we expand to < 2.4?

Reply:
ehh. I'd stick with 2.4 for now - see if we get any complaints